saleor/graphql/api.py

```diff
@@ -47,9 +48,9 @@ class Meta:
         model = Product
         interfaces = (relay.Node, DjangoPkInterface)
 
-    def resolve_image_url(self, args, context, info):
-        image = self.images.first()
-        return image.image.crop['400x400'].url if image else None
+    def resolve_thumbnail_url(self, args, context, info):
+        image_obj = self.images.first()
+        return get_thumbnail(image_obj.image if image_obj else None, '400x400')
```

Comment:
There is also a template tag that fetches first product image. If there's no image it returns a placeholder.